Assets/HoloToolkit/Utilities/Scripts/Editor/SolverHandlerEditor.cs

```diff
@@ -12,24 +12,45 @@ public class SolverHandlerEditor : ControllerFinderEditor
     {
         private SerializedProperty trackedObjectReferenceProperty;
         private SerializedProperty transformTargetProperty;
+        private SerializedProperty additionalOffsetProperty;
+        private SerializedProperty additionalRotationProperty;
+        SolverHandler solverHandler;
 
         protected override void OnEnable()
         {
             base.OnEnable();
 
             trackedObjectReferenceProperty = serializedObject.FindProperty("trackedObjectToReference");
             transformTargetProperty = serializedObject.FindProperty("transformTarget");
+            additionalOffsetProperty = serializedObject.FindProperty("additionalOffset");
+            additionalRotationProperty = serializedObject.FindProperty("additionalRotation");
+
+            solverHandler = target as SolverHandler;
         }
 
         public override void OnInspectorGUI()
         {
             serializedObject.Update();
             EditorGUILayout.Space();
 
+            EditorGUI.BeginChangeCheck();
             EditorGUILayout.PropertyField(trackedObjectReferenceProperty);
-            EditorGUILayout.PropertyField(transformTargetProperty);
+            bool trackedObjectChanged = EditorGUI.EndChangeCheck();
+            
+            if (!Application.isPlaying)
+            {
+                EditorGUILayout.PropertyField(additionalOffsetProperty);
+                EditorGUILayout.PropertyField(additionalRotationProperty);
+            }
+
+            EditorGUILayout.PropertyField(transformTargetProperty, new GUIContent("Tracked Transform"));
```

Comment:
let's cache the gui content so we're not creating a new one each gui update

Reply:
or just rename the field in the base class to `trackedTransform`

Reply:
Won't renaming that field break existing data?